shared/actions/push.js

```diff
@@ -67,6 +68,10 @@ function * pushNotificationSaga (notification: PushNotification): SagaGenerator<
       return
     }
 
+    // If we're going to a pressed push notification, we don't *also* want to
+    // restore the previous stored initial tab.
+    yield put(setInitialTab(null))
```

Comment:
this'll still race. we should that we came in w/ a push into the store and just ignore the initial tab. it's possible we go through this flow, set initialTab to be null, but the callback to loading that value is still in flight.